packages/rx_bloc_cli/mason_templates/bricks/feature_cicd_fastlane/__brick__/{{#cicd_github}}.github{{/cicd_github}}/workflows/fastlane_android_custom_build_and_deploy.yaml

```diff
@@ -53,6 +53,9 @@ jobs:
         with:
           flutter-version: ${{ inputs.flutter_version }}
           channel: "stable"
+          cache: true
```

Comment:
Please avoid any non-todoapp related changes

Reply:
As long as I see it's coming from here: https://github.com/Prime-Holding/rx_bloc/blob/feat/remove-credentials-after-build-deploy/.github/workflows/fastlane_android_custom_build_and_deploy.yaml
@JvnSlv is it possible that this is from the division app fixes?

Reply:
@DDavidPrime please review this change

Reply:
> As long as I see, it's coming from here: https://github.com/Prime-Holding/rx_bloc/blob/feat/remove-credentials-after-build-deploy/.github/workflows/fastlane_android_custom_build_and_deploy.yaml @JvnSlv is it possible that this is from the division app fixes?

I don't think so. Only changes that I have done were related to the example projects, I didn't change anything rx_bloc_cli related

Reply:
As it turns out, the change is actually merged from `develop` where we removed the caching for android/ios build phases. We discussed this with @StanevPrime and agreed that it does not bring any benefits to the build/deploy workflows as it only adds additional minutes for caching (which is not used when in combination with tags).

Reply:
The change has been reverted.